app/src/main/java/com/bogdan/codeforceswatcher/features/users/redux/requests/UsersRequests.kt

```diff
@@ -46,7 +48,9 @@ class UsersRequests {
                             difference.add(Pair(user.handle, delta))
                         }
                     }
-                    DatabaseClient.userDao.update(user)
+                    withContext(Dispatchers.IO) {
```

Comment:
Moving DB operations to `Dispatchers.IO` actually good in terms of performance, but complicates code. So not sure if we need it. If it was a part of your tries to resolve UI bugs, think one more if you want to write it like this everywhere.